packages/atomic/create-atomic/src/list-pages.ts

```diff
@@ -0,0 +1,8 @@
+import {PlatformClient} from '@coveord/platform-client';
+
+export async function listSearchPagesOptions(client: PlatformClient) {
+  return (await client.searchInterfaces.list({})).items.map((page) => ({
```

Comment:
What are the default `list` params in relation to pagination ?

Could it cause issues if you have too much search pages configured in a single org, that we would not be able to pick it up ?

Reply:
It's 150 per page with page=0. We could put a pagination system to try to fetch more than 150, but it might be unrealistic?